RussianFilter/sections/specific.txt

```diff
@@ -182,6 +182,10 @@ bizlida.by##.big-poster
 portal.lviv.ua###kinotvukrnet
 08sportbar.online,08sportbar.net,sport24hd.net,sport24hd.com##body > div[style*="position: absolute;"][style*="z-index: 9999;"]
 ||emb.apl14.me/js/pop.js
+||gazetaby.info/assets/resized/300-200-crop-t/uploads/bunners/
```

Comment:
Check again. This rule does not cover one banner image.
`||gazetaby.info/*uploads/bunners/`